src/pykeen/models/unimodal/rescal.py

```diff
@@ -70,12 +71,16 @@ def __init__(
         """
         super().__init__(
             triples_factory=triples_factory,
-            embedding_dim=embedding_dim,
-            relation_dim=embedding_dim ** 2,  # d x d matrices
             loss=loss,
             preferred_device=preferred_device,
             random_seed=random_seed,
             regularizer=regularizer,
+            entity_representations=EmbeddingSpecification(
+                embedding_dim=embedding_dim,
+            ),
+            relation_representations=EmbeddingSpecification(
+                embedding_dim=embedding_dim ** 2,  # d x d matrices
```

Comment:
we can also use `shape` instead of `embedding_dim` right? This would be even more explicit than this comment.

Reply:
yes. that will be next PR, since that changes how everything works